MzmlReader/MzmlReader.cs

```diff
@@ -40,12 +39,14 @@ public MzmlReader()
         public Run LoadMzml(string path, double massTolerance, bool storeScansInMemory, string irtPath)
         {
             Run run = new Run();
+            run.AnalysisSettings.MassTolerance = massTolerance;

```

Comment:
Don't set this here. LoadMzml should accept an AnalysisSettings object as a variable and consider adding it to Run in the Run constructor.